docs/detections/rules-ui-create.asciidoc

```diff
@@ -820,3 +820,20 @@ TIP: Avoid setting long time ranges with short rule intervals, or the rule previ
 
 * To close the preview, click the *Rule preview* button again.
 
+[discrete]
+[[debug-rule-queries]]
+==== Debug rule queries (optional)
```

Comment:
I wanted to make the title goal-oriented, but maybe that's not the right direction, especially since this feature only provides the query for debugging. How about this instead:

```suggestion
==== Inspect {es} queries sent by your rule (optional)
```